src/main/java/host_room/HostRoomPresenter.java

```diff
@@ -0,0 +1,26 @@
+package host_room;
+
+import ui.View;
+
+import javax.swing.JOptionPane;
+public class HostRoomPresenter implements HostRoomOutputBoundary {
+    private View view;
```

Comment:
is it okay to directly initialize abstract class?

Reply:
I don't think we're doing that here? We initialize the children of `View` in our `main` method and pass those in here.